src/content/docs/new-relic-solutions/business-observability/install-pathpoint.mdx

```diff
@@ -10,9 +10,10 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-import installPathpoint from 'images/pathpoint_screenshot_install-pathpoint.webp' 
+import installPathpoint from 'images/pathpoint_v2_screenshot_install_pathpoint.webp' 
+  
 
-New Relic Pathpoint can be integrated with almost any New Relic account to provide comprehensive business journey observability. To get started, it's important to have telemetry data that is relevant to your organization. This can include metrics, events, logs, or traces. While APM events and logs are a common starting point for Pathpoint, the choice of telemetry data is entirely up to you. 
+New Relic Pathpoint can be integrated with almost any New Relic account to provide comprehensive business process observability. To get started, it's important to have telemetry data that is relevant to your organization. This can include metrics, events, logs, or traces. While APM events and logs are a common starting point for Pathpoint, the choice of telemetry data is entirely up to you. 
```

Comment:
```suggestion
New Relic Pathpoint can be integrated with almost any New Relic account to provide comprehensive business process observability. To get started, it's important to have telemetry data that's relevant to your organization. This can include metrics, events, logs, or traces. While APM events and logs are a common starting point for Pathpoint, the choice of telemetry data is entirely up to you. 
```